dance.css

```diff
@@ -126,11 +128,14 @@ data {
 }
 
 ol {
+  contain: content;
+  display: flow-root;
```

Comment:
the clearing technique works on its own but this is backup and head's up about floating